src/components/drawers/EventEditDrawer.tsx

```diff
@@ -0,0 +1,82 @@
+import { formatDate } from '@/utils/formatDate'
+import {
+  Drawer,
+  DrawerBody,
+  DrawerContent,
+  DrawerHeader,
+  DrawerOverlay,
+} from '@chakra-ui/react'
+import { EventForm } from '../forms/EventForm'
+import {
+  IConfigTaxonomyCCLW,
+  IConfigTaxonomyUNFCCC,
+  IEvent,
+} from '@/interfaces'
+
+type TProps = {
+  editingEvent?: IEvent
+  loadedFamilyId: string
+  organisation: string
+  canModify: (
+    orgName: string | null,
+    isSuperUser: boolean,
+    userAccess:
```

Comment:
Can we manage this type in a separate file and import it here? This type definition feels quite clunky and we will need it elsewhere

Reply:
Changes implemented